TP-Sampler/src/utils.js

```diff
@@ -77,7 +77,11 @@ function parseSpecifier(spec, rangeWord) {
         arr = arr.concat(seq);
       }
     } else {
-      arr.push(item);
+      if (item !== "") {
```

Comment:
You could also add `item = item.trim()` and `if (item.length > 0) { arr.push(item) }`.